UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.UnitTests/Services/FssEventProcessorTest.cs

```diff
@@ -83,7 +83,6 @@ public async Task WhenInvalidBusinessUnitInRequest_ThenReceiveSuccessfulResponse
 
             ExternalNotificationServiceProcessResponse result = await _fssEventProcessor.Process(_fakeCustomCloudEvent, CorrelationId, cancellationToken);
 
-            Assert.AreEqual("Invalid business unit in an event.", result.Errors.Single().Description);
             Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
         }
```

Comment:
I think we actually need to add the important assertion that no notification is published when it recieves an FSS event for an unrecognised business unit.

```suggestion
    A.CallTo(() => _fakeAzureEventGridDomainService.PublishEventAsync(A<CloudEvent>.Ignored, A<string>.Ignored, A<CancellationToken>.Ignored)).MustNotHaveHappened();
}
```

Reply:
Yes, good shout, done.